homeassistant/components/sensibo/translations/en.json

```diff
@@ -31,6 +31,38 @@
         }
     },
     "entity": {
+        "climate": {
+            "climate_device": {
+                "state_attributes": {
+                    "fan_mode": {
+                        "state": {
+                            "auto": "Auto",
+                            "high": "High",
+                            "low": "Low",
+                            "medium": "Medium",
+                            "medium_high": "Medium high",
+                            "quiet": "Quiet"
```

Comment:
These seems to be not mapped?

As in, I cannot find the list of possible states in Home Assistant mapped to the integration library?

Reply:
There is no official list of possible states so the library is just passing through what comes from the api.
But unofficially I have seen a list from the maker that these are the current possible states for swing and fan.

I would not think I need a fixed list of states?

Reply:
Well, the issue with not mapping it, is that we also won't know about things missing. When mapping and blowing up with an unexpected state, it makes it possible to extend the list as needed.

Reply:
I see the point. Perhaps better to implement a list of possible states and extend when new things appear.
Would you propose to just create a constant with a list in the integration of possible states or would it be better to have it in the library and use it in the integration?

Reply:
> Would you propose to just create a constant with a list in the integration of possible states

You'd need the mapping in HA anyways, so keeping it here is fine.

Reply:
Available `fan_modes` and `swing_modes` is now implemented in this PR.
I'll update the PR description as well